package.json

```diff
@@ -35,11 +35,24 @@
   "main": "./out/extension.js",
   "l10n": "./l10n",
   "contributes": {
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "salesforcedx-vscode-offline-app.configureLintingTools",
+          "when": "sfdx_project_opened"
+        }
+      ]
+    },
     "commands": [
       {
         "command": "salesforcedx-vscode-offline-app.onboardingWizard",
         "title": "%extension.commands.config-wizard.title%",
         "category": "%extension.commands.config-wizard.category%"
+      },
+      {
+        "command": "salesforcedx-vscode-offline-app.configureLintingTools",
+        "title": "%extension.commands.config-linting-tools.title%",
+        "category": "%extension.commands.config-wizard.category%"
```

Comment:
Hmmm, I'm not sure about this. In theory this doesn't just apply to the Starter Kit—or really at all, because the Starter Kit already has the linting rules configured out of the box. I wonder if we should define a new category, perhaps called "Salesforce Mobile Offline"?